studio/schemas/objects/sections/contact-box.ts

```diff
@@ -0,0 +1,119 @@
+import { defineField } from "sanity";
+
+import { StringInputWithCharacterCount } from "studio/components/stringInputWithCharacterCount/StringInputWithCharacterCount";
+import { subtitleID, titleID } from "studio/schemas/fields/text";
+import { firstTranslation } from "studio/utils/i18n";
+
+export const contactBoxID = "contactBox";
+
+export enum ContactBoxBackground {
+  Dark = "dark",
+  Light = "light",
+}
+
+const backgroundOptions = [
+  { title: "Dark", value: ContactBoxBackground.Dark },
+  { title: "Light", value: ContactBoxBackground.Light },
+];
+
+export const contactBox = defineField({
+  name: contactBoxID,
+  title: "Contact Box",
+  type: "object",
+  fields: [
+    {
+      name: titleID.basic,
+      type: "internationalizedArrayString",
+      title: "Title",
+      description:
+        "Enter the primary title that will be displayed as large text in contact box.",
+    },
+    {
+      name: subtitleID.optional,
+      type: "internationalizedArrayString",
+      title: "Subtitle",
+      description: "Extra context under the main title.",
+    },
+    {
+      name: "background",
+      title: "Background",
+      type: "string",
+      description:
+        "Select the whether the contact box should have light or dark background.",
+      options: {
+        list: backgroundOptions,
+        layout: "radio",
+      },
+      initialValue: ContactBoxBackground.Dark,
+    },
+    {
+      name: "contactPoints",
+      title: "List of people to contact",
+      description: "List of tags and people to contact within those tags.",
```

Comment:
What does `tags` mean in this case?

Reply:
Tag in this case is the text used in buttons/tag to identify. Wanting to have it generic to support things like "Trondheim", "Oslo" or "Sales", "Recruitment" etc

Reply:
Nice! Perhaps add some additional explanation or an example to the description so it is easier to understand for the content creators